examples/navigator/custom_tabbar/tab_1.xml.njk

```diff
@@ -0,0 +1,42 @@
+---
+title: "Tab Navigator - Tab 1"
+permalink: "/navigator/custom_tabbar/tab_1.xml"
+---
+
+{% extends 'base.xml.njk' %}
+{% block container %}
+  <view scroll="true">
+    <text style="Description">This is the content for tab 1.</text>
+  </view>
+{% endblock %}
+
+{% block footer %}
+  <footer>
+    <view id="footer" style="Footer">
```

Comment:
do we need a separate nested container, or could `footer` hold the props `id` and `style`?